packages/teleport-plugin-react-styled-jsx/src/index.ts

```diff
@@ -59,13 +59,19 @@ export const createPlugin: ComponentPluginFactory<StyledJSXConfig> = (config) =>
     }
 
     const jsxASTNodeReference = generateStyledJSXTag(styleJSXString.join('\n'))
-    const rootJSXNode = jsxNodesLookup[(node.content as UIDLElement).key] // TODO: Check for other types
-
     // We have the ability to insert the tag into the existig JSX structure, or do something else with it.
     // Here we take the JSX <style> tag and we insert it as the last child of the JSX structure
     // inside the React Component
-    rootJSXNode.children.push(jsxASTNodeReference)
+    let rootJSXNode
```

Comment:
It's easier to compute the rootJSXNode as the result of `traverseUIDLForRootElmNode` then you don't need to do the cast here nor the if statement